### DISCUSSION TITLE
Attachement not visable

### DISCUSSION THREAD

**Discussion by CornAndBerry** [postalserver/postal#1533] [Help with using Postal]
When I go to my messages I can't see my attachments anymore. What could be the problem here? 

![image](https://user-images.githubusercontent.com/84896713/128710792-57ac1049-1f0a-45a9-85dc-4c5816bd1e1d.png)

**Comment by willpower232** [comment]
If you have a look at the Settings tab, you should see what the retention settings are for this server.

**Reply by deefdragon** [reply]
I am using postal to route my DKIM messages to a server that parses and reports them to me. When postal gets the messages directly, no attachment is displayed and I do not get the attachment in the webhook. When I forward the email to postal, the UI displays the attachment and the webhook gets the attachment. 

The strange thing is that for the missing instances, downloading the raw email shows the attachment. The attachment _exists_ its just not getting picked up in this one case for postal.

Im wondering if this is an issue case because the DKIM messages have no HTML or text body?
